boards/arm/stm32/stm32f4discovery/configs/wifi/defconfig

```diff
@@ -55,12 +55,9 @@ CONFIG_NETUTILS_HTTPD_SENDFILE=y
 CONFIG_NETUTILS_TELNETD=y
 CONFIG_NETUTILS_WEBCLIENT=y
 CONFIG_NETUTILS_WEBSERVER=y
-CONFIG_NET_BROADCAST=y
-CONFIG_NET_ICMP_NO_STACK=y
-CONFIG_NET_TCP_NO_STACK=y
-CONFIG_NET_UDP_NO_STACK=y
 CONFIG_NET_USRSOCK=y
 CONFIG_NET_USRSOCK_CONNS=16
+CONFIG_NET_USRSOCK_ICMP=y
 CONFIG_NET_USRSOCK_UDP=y
```

Comment:
@xiaoxiang781216
Because the GS2200M does not support ICMP, I think it should be removed.

Reply:
Done.